src/callbacks.rs

```diff
@@ -117,8 +117,22 @@ pub extern fn view_request_move(view: WlcView, _dest: &Point) {
     }
 }
 
-pub extern fn view_request_resize(_view: WlcView, _edge: ResizeEdge, _location: &Point) {
-
+pub extern fn view_request_resize(view: WlcView, edge: ResizeEdge, point: &Point) {
+    if let Ok(mut tree) = try_lock_tree() {
+        match try_lock_action() {
+            Ok(guard) => {
+                if guard.is_some() {
+                    if let Some(id) = tree.lookup_view(view) {
+                        if let Err(err) = tree.resize_container(id, edge, *point) {
+                            error!("Context: Trying to resize a container for the user \
```

Comment:
> the point of this error was to help the debugger (eg us) understand what the user was doing without having to mentally recreate the scene. 

I'm not sure that warrants too much extra information in the log - we already have the line in the source code, so we know it's in the callbacks, and it's the tree failing to resize a container. The only way we're _really_ going to know what the user was doing without having to mentally recreate the scene is to `trace!`-log every single user interaction, and I think our layout code is stable enough that we shouldn't have to do that in production code. We should either try immediately logging the state of the tree + information (values of variables, not a description of the code) about the action being called, or only enable `trace!` logs for debug builds and try to recreate errors on our machines in response to bug reports.